docs/applications/containers/how-to-install-docker-images.md

```diff
@@ -0,0 +1,104 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+contributor:
+  name: Jack Wallen
+  link: https://twitter.com/jlwallen
+description: 'An introduction to using Docker, containers, and dockerfiles on your Linode.'
+keywords: 'docker,container,dockerfile'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: Wednesday, June 28, 2017
+modified_by:
+  name: Linode
+published: 'Wednesday, June 28, 2017'
+title: 'How to Install Docker and Pull Images '
+external_resources:
+ - '[Docker Docs](http://docs.docker.com/)'
+ - '[Docker Try it Tutorial](https://www.docker.com/tryit/)'
+ - '[Docker Hub](https://hub.docker.com/)'
+ 
```

Comment:
Why the empty space?

![bitmoji](https://render.bitstrips.com/v2/cpanel/10220248-243506170_5-s4-v1.png?transparent=1&palette=1&width=246)